testing/test_packages/conda_pack_test_lib3/conda.recipe/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: conda_pack_test_lib3
+  version: 0.0.2
+
+source:
+  path: ../
+
+build:
+  number: 1
+  noarch: generic
```

Comment:
this should be "noarch: posix" if such thing existed. We can remove this line if we rebuild the package for Linux so that Travis can use it.